images/linux/scripts/installers/1604/hosted-tool-cache.sh

```diff
@@ -31,14 +31,23 @@ DocumentInstalledItem "Python:"
 pythons=$(ls $AGENT_TOOLSDIRECTORY/Python)
 for python in $pythons; do
 	DocumentInstalledItemIndent "Python $python"
+	# Add symlinks for python, python2 and python3 so that Python can be used out of the box without setup-python
+	if [ ${python:0:3} == "2.7" ] ; then
+		# add python 2.7 for python and python2
+		ln -s $AGENT_TOOLSDIRECTORY/Python/$python/x64/bin/python /usr/local/bin/python
+		ln -s $AGENT_TOOLSDIRECTORY/Python/$python/x64/bin/python /usr/local/bin/python2
```

Comment:
We'll definitely want `pip` and `pip3` in `PATH`, along with the various command-line tools that come with Python.  I think the best thing to do here would just be to add the tool cache directories to `PATH` rather than create symlinks.

Reply:
You could also just add every tool cache directory to `PATH` -- `ls` should guarantee us a sorted order here.  That would make commans like `tox -e py37` work on Actions, which I think is something we should enable.